.changeset/hot-jobs-return.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/ui-extensions': patch
+'@shopify/ui-extensions-react': patch
```

Comment:
nothing is used yet, I probably wouldn't worry too much about this being a patch?